tests/tests_lite/strategies/launchers/test_multiprocessing.py

```diff
@@ -36,7 +36,9 @@ def test_multiprocessing_launcher_forking_on_unsupported_platform(_):
 
 @pytest.mark.parametrize("start_method", ["spawn", "fork"])
 @mock.patch("lightning_lite.strategies.launchers.multiprocessing.mp")
-def test_multiprocessing_launcher_start_method(mp_mock, start_method):
+# we're just testing invocation so we bypass the CUDA check here to avoid failure if another test has initialized CUDA
+@mock.patch.object(torch.cuda, "is_initialized", return_value=False)
+def test_multiprocessing_launcher_start_method(_, mp_mock, start_method):
```

Comment:
I had a fix in a separate branch already: https://github.com/Lightning-AI/lightning/pull/14550/commits/b6743ad125dba6721206cd4462072c9e903b58fb

Marking it as standalone sounds like a better fix to me. Wdyt @awaelchli?

We can copy the change over to this PR

Reply:
thanks for reviewing @carmocca,  porting over your standalone approach SGTM